### PR TITLE
added init script with test password

### PR DESCRIPTION
**Pull request by okwme**
Closes: #XXX

## Description

Added an init.sh script that uses the `test` `keyring-backed` because multiple users have had trouble unlocking the RPC endpoint.

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+rm -rf ~/.emintcli
+rm -rf ~/.emintd
```
Comment (fedekunze) [inline init.sh:3]: I'd run `make install` after this just to be sure the user has the latest binary

**Review by fedekunze — COMMENTED** [review]
Billyyyyyyy! 🙌 
Changes are @okwme. Can you add a note on the README to run the script?

File: init.sh (hunk 1 above)
Comment (okwme) [reply init.sh:3]: 👍

**Comment by okwme** [comment]
Made some updates to the README.md, although I'm suggesting to use the `keyring-backend` profile `test`. I've spoken to multiple mac users who were having problems getting their RPC endpoint unlocked. I'm not sure if test fixes that problem or not, I don't have access to a mac to confirm atm.

**Comment by fedekunze** [comment]
@okwme did you push your changes to the README? I only see the script

**Comment by okwme** [comment]
@fedekunze sorry forgot to push, just did now

**Review by fedekunze — APPROVED** [review]
tested ACK

**Review by austinabell — APPROVED** [review]
Yeah, I should have updated the README when the keyring changes came in, I was just hopeful that usage would be cleaner by the stable release

edit:
> Made some updates to the README.md, although I'm suggesting to use the `keyring-backend` profile `test`. I've spoken to multiple mac users who were having problems getting their RPC endpoint unlocked. I'm not sure if test fixes that problem or not, I don't have access to a mac to confirm atm.

Yeah on a mac it works, but it's not clean to use. I can see how people would run into issues

**Comment by fedekunze** [comment]
@GregTheGreek, @ColinSchwarz or @austinabell  can you merge the PR? it seems that external contributors don't have permissions to run CircleCI

**Pull request merged**